src/redux/businessPages/businessPages.actions.js

```diff
@@ -0,0 +1,28 @@
+import {
+  GET_BUSINESS_PAGE_BY_CODE,
+  SET_LOADING,
+  SET_BUSINESS_PAGE
+} from './businessPages.types';
+
+const setBusinessPage = (businessPage) => ({
+  type: SET_BUSINESS_PAGE,
+  payload: {
+    businessPage,
+    key: businessPage.code
+      .split('-')
+      .map((val, idx) => (idx ? val[0].toUpperCase() + val.slice(1) : val))
```

Comment:
Please, rewrite `idx ?` to `idx == 0 ?`, it is much easier to understand if you need condition when `idx` is equal to 0